projects/fhi-angular-highcharts/README.md

```diff
@@ -158,6 +161,15 @@ FhiDiagramSerieData is a custum type for FHI Angular Highcharts, but it is based
 | `label`  | `string` | -       | yes      | Description of the flag, ie. _Anonymized_ |
 | `symbol` | `string` | -       | yes      | Symbol representing the flag, ie. `:`. |
 
+### Type FhiDiagramUnit
+
+| Property   | Type     | Default | Required | Description |
+| ---------- | -------- | ------- | -------- | ----------- |
+| `decimals` | `number` | -       | no       | The decimal count. If not set the decimal count same as in data. Count is **limited to 14** decimals due to loss of precision at runtime above 14. A warning will be given in the consol if more than 14 decimals. |
```

Comment:
Is 'id' missing here?

Reply:
Removed the id instead, since it's not in use yet.